src/components/reusable/Card.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import styled from "styled-components";
+import PrimaryButton from "./PrimaryButton";
+import { theme } from "../../theme";
+
+export default function Card({ imageSource, title, leftDescription }) {
+  return (
+    <CardStyled>
+      <img src={imageSource} alt="" />
+      <div className="text-info">
+        <strong className="title">{title}</strong>
+        <div className="description">
+          <p className="left-description">{leftDescription}</p>
+          <PrimaryButton label={"Ajouter"} className="card-primary-button" />
+        </div>
+      </div>
+    </CardStyled>
+  );
+}
+
+const CardStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;

```

Comment:
à check avec ton design système, objet "spacing" ( si fichier thème à jour )
```suggestion
${theme.spacing.md};
```